manifests/l/LukaszSwiszcz/Winpinator/0.1.2/LukaszSwiszcz.Winpinator.locale.en-US.yaml

```diff
@@ -0,0 +1,29 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: LukaszSwiszcz.Winpinator
+PackageVersion: 0.1.2
+PackageLocale: en-US
+Publisher: Lukasz Swiszcz

```

Comment:
```suggestion
Publisher: Łukasz Świszcz
```
Shouldn't we also write this with the correct alphabets and match it with Control Panel as well?

Reply:
Right! I should have double checked in control panel, the third-party program I used to check APR shows normal text.
![image](https://user-images.githubusercontent.com/71691514/207279677-791f73b1-eaa7-4e4c-b1aa-3f84d3ece519.png)